### PR TITLE
Generates_score removal + some extra refactoring

### PR DESCRIPTION
**Pull request by OlafSzmidt**

### CODE REVIEW

File: aimmo-game-worker/test-turn.py [hunk 1]
```diff
@@ -8,17 +8,14 @@
...
-                False, 'avatar': avatar, 'pickup': {'health_restored': 3}},
+            {'location': {'x': 0, 'y': 0}, 'habitable': True, 'avatar': avatar, 'pickup': None},
+            {'location': {'x': 1, 'y': 0}, 'habitable': False, 'avatar': None, 'pickup': None},
+            {'location': {'x': -1, 'y': 0}, 'habitable': True, 'avatar': avatar, 'pickup': {'health_restored': 3}},
         ],
     }
 }
 
-print 'Posting:', api_data
+print('Posting:', api_data)
```
Comment (danalex97) [inline aimmo-game-worker/test-turn.py]: Not necessarily to the PR, maybe we should use logs here and move the package to the tests folder.

File: aimmo-game/tests/test_simulation/maps.py [hunk 2]
```diff
@@ -23,11 +23,10 @@ def serialise(self):
 
 
 class MockCell(Cell):
-    def __init__(self, location=1, habitable=True, generates_score=False,
-                 avatar=None, pickup=None, name=None, actions=[]):
+    def __init__(self, location=1, habitable=True, avatar=None,
+                 pickup=None, name=None, actions=[]):
```
Comment (danalex97) [inline aimmo-game/tests/test_simulation/maps.py]: I think you changed `actions` here(the bug PR from yesterday), maybe you should merge/rebase?

**Comment by OlafSzmidt** [comment]
@danalex97 Done, I think ;)

**Comment by danalex97** [comment]
Btw, do we still use `test-initialize` and `test-turn`?

**Comment by OlafSzmidt** [comment]
So I tried moving them to tests, but the `game-worker-tests` failed because these files are manually specified somewhere else for testing. I think it's `aimmo-game-worker/aimmo_game_worker.egg-info/SOURCES.txt`. Hmmm.

**Comment by danalex97** [comment]
Eggs are automatically generated using `setup.py`, so I think you can safely remove them.

**Pull request merged**